Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -30,109 +29,93 @@ class OutputHelper
 
     void PrintLoadingInfo(const std::wstring& modelPath) const
     {
-        if (!m_silent)
-        {
-            wprintf(L"Loading model (path = %s)...\n", modelPath.c_str());
-        }
+        wprintf(L"Loading model (path = %s)...\n", modelPath.c_str());
     }
 
-    void PrintBindingInfo(uint32_t iteration, DeviceType deviceType, InputBindingType inputBindingType, InputDataType inputDataType, DeviceCreationLocation deviceCreationLocation) const
+    void PrintBindingInfo(uint32_t iteration, DeviceType deviceType, InputBindingType inputBindingType, InputDataType inputDataType, DeviceCreationLocation deviceCreationLocation, const std::string& status) const
     {
-        if (!m_silent)
-        {
-            printf(
-                "Binding (device = %s, iteration = %d, inputBinding = %s, inputDataType = %s, deviceCreationLocation = %s)...",
-                TypeHelper::Stringify(deviceType).c_str(),
-                iteration,
-                TypeHelper::Stringify(inputBindingType).c_str(),
-                TypeHelper::Stringify(inputDataType).c_str(),
-                TypeHelper::Stringify(deviceCreationLocation).c_str()
-            );
-        }
+        printf(
+            "Binding (device = %s, iteration = %d, inputBinding = %s, inputDataType = %s, deviceCreationLocation = %s)...%s\n",
+            TypeHelper::Stringify(deviceType).c_str(),
+            iteration,
+            TypeHelper::Stringify(inputBindingType).c_str(),
+            TypeHelper::Stringify(inputDataType).c_str(),
+            TypeHelper::Stringify(deviceCreationLocation).c_str(),
+            status.c_str()
+        );
     }
 
-    void PrintEvaluatingInfo(uint32_t iteration, DeviceType deviceType, InputBindingType inputBindingType, InputDataType inputDataType, DeviceCreationLocation deviceCreationLocation) const
+    void PrintEvaluatingInfo(uint32_t iteration, DeviceType deviceType, InputBindingType inputBindingType, InputDataType inputDataType, DeviceCreationLocation deviceCreationLocation, const std::string &status) const
     {
-        if (!m_silent)
-        {
-            printf(
-                "Evaluating (device = %s, iteration = %d, inputBinding = %s, inputDataType = %s, deviceCreationLocation = %s)...",
-                TypeHelper::Stringify(deviceType).c_str(),
-                iteration,
-                TypeHelper::Stringify(inputBindingType).c_str(),
-                TypeHelper::Stringify(inputDataType).c_str(),
-                TypeHelper::Stringify(deviceCreationLocation).c_str()
-            );
-        }
+        printf(
+            "Evaluating (device = %s, iteration = %d, inputBinding = %s, inputDataType = %s, deviceCreationLocation = %s)...%s",
```

Comment:
```suggestion
            "Evaluating (device = %s, iteration = %d, inputBinding = %s, inputDataType = %s, deviceCreationLocation = %s)...%s\n",
```
Need to add \n newline here. If we run without -perf flag, and with multiple configurations, the next configuration's binding console output will be on the same line as the previous evaluate console output